fourkitchens/sous-drupal-distro
# Emulsify Design System update

## What does it do?

- Changes to new `emulsify_design_system` instead of `emulsify`
- Fixes references to match new "Emulsify Design System" language instead of "Emulsify with Gatsby"

## How to test

1. Run `composer install`
2. Verify `unified_twig_extensions` is not in the modules directory but Emulsify's new dependency `emulsify_twig` is.
3. Verify code changes look accurate.

## Questions 

Should `core` and/or `web` be gitignored?
